Successfully updated:
Created Frame node MPsoyWiXW
Created Frame node RonU5918V
Created Text node L3mGIxayk
Created Text node NtASvmP9k
Created Frame node T5mwJ0Vqo
Updated attributes for node MPsoyWiXW
Updated attributes for node RonU5918V
Updated text for node L3mGIxayk
Updated attributes for node L3mGIxayk
Updated text for node NtASvmP9k
Updated attributes for node NtASvmP9k
Updated attributes for node T5mwJ0Vqo
Reordered node RonU5918V within parent MPsoyWiXW to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -46,41 +46,40 @@
       stackDirection="horizontal"
       stackDistribution="center"
       stackAlignment="center"
   >
     <NavigationFlyout
         nodeId="BwfPOa8WY"
         width="100%"
         height="fit-content"
         <!-- the component id this instance uses -->
         componentId="sCy40XUNu"
         <!-- 'Desktop' is QOfAnzBxh, 'Tablet' is P1v1ihspX, 'Phone' is PnuvdVM1J, 'Desktop Flyout' is arFY89os4, 'Tablet Flyout' is qGUh0LbE8, 'Phone Flyout' is uWt60osFn, 'Tablet close' is wrwRZcibf, 'Tablet  Flyout close' is WkaXeslVy, 'Phone close' is NTfXwQ4NI, 'Phone Flyout close' is DrAxuXYke -->
         variant="arFY89os4"
      />
   </NavigationFlyoutFixed>
   <Navigation
       nodeId="P8AJj3Q2E"
       width="1fr"
       height="fit-content"
       <!-- the component id this instance uses -->
       componentId="sCy40XUNu"
-      <!-- 'Desktop' is QOfAnzBxh, 'Tablet' is P1v1ihspX, 'Phone' is PnuvdVM1J, 'Desktop Flyout' is arFY89os4, 'Tablet Flyout' is qGUh0LbE8, 'Phone Flyout' is uWt60osFn, 'Tablet close' is wrwRZcibf, 'Tablet  Flyout close' is WkaXeslVy, 'Phone close' is NTfXwQ4NI, 'Phone Flyout close' is DrAxuXYke -->
       variant="QOfAnzBxh"
    />
   <HeroSection
       nodeId="qKDdG3FQZ"
       width="1fr"
       height="fit-content"
       layout="stack"
       padding="8px"
       stackDirection="vertical"
       stackDistribution="start"
       stackAlignment="center"
   >
     <Container
         nodeId="rUPekfMHT"
         width="1fr"
         height="fit-content"
         maxWidth="1152px"
         <!-- background color string or project color style path (if starts with /) -->
         backgroundColor="white"
         borderRadius="32px"
@@ -3061,20 +3060,73 @@
       Header Text
     </HeaderText>
   </Frame>
   <BodyContentGoesHere
       nodeId="XAYOrGFUB"
       width="fit-content"
       height="fit-content"
       font="Inter"
   >
     Body content goes here
   </BodyContentGoesHere>
   <Frame
       nodeId="V3VqY3j0F"
       width="100%"
       height="100px"
       <!-- background color string or project color style path (if starts with /) -->
       backgroundColor="rgb(255, 255, 255)"
       borderRadius="4px"
    />
 </Stack>
+<Stack
+    nodeId="MPsoyWiXW"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="RonU5918V"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="L3mGIxayk"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="NtASvmP9k"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="T5mwJ0Vqo"
+      width="100%"
+      height="1fr"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.